index.html

```diff
@@ -186,41 +187,83 @@ <h1 class="pycon-journey">JOURNEY</h1>
                       <h2 class="animated wow bounceInLeft" data-wow-delay="0.2s" data-wow-duration="2s">Keynote<br><span> Speakers</span></h2>
                       <h1 class="pycon-speaker">SPEAKERS</h1>
                         <div class="row pycon2018_spe">
-                          <div class="col-md-3 col-md-offset-1 text-center">
-                             <img src="img/border1.png" class="border1" alt="border">
-                             <img src="img/armin-ronacher.png" alt="image">
-                             <h3 class="text-right">Armin Ronacher</h3>
-                             <h5 class="text-right">Creator of Flask</h5>
-                             <h5 class="text-right">Software Engineer, Sentry.io</h5>
+                          <div class="col-md-4 text-center">
```

Comment:
Use `col-md-3` and `col-md-offset-1` for the first block. This is intentional since we will be having one more keynote speaker

Reply:
Cool. will fix it.